### ISSUE TITLE
msearch missing version and timeout

### ISSUE THREAD

**Issue by prafsoni** [elastic/elasticsearch-java#161]
We could do 
```
POST test_index/_msearch
{}
{ "version": true, "query": { "match_all": {} } , "timeout": "10s"}
```
But the client doesn't allow to set flag `version: true` and timeout in msearch request body making in impossible to get version for hits in response

elasticsearch-java version: 7.17.0

**Comment by sothawo** [comment]
+1 for the `version` parameter, `seq_no_primary_term` cannot be set either

**Comment by swallez** [comment]
This has been fixed in PR #292, backported to version 8.2 in PR #293 and to version 7.17 in PR #294.
